azurelinuxagent/common/osutil/freebsd.py

```diff
@@ -93,6 +97,192 @@ def get_if_mac(self, ifname):
     def get_first_if(self):
         return self._get_net_info()[:2]
 
+    @staticmethod
+    def read_route_table():
+        """
+        Return a list of strings comprising the route table, including column headers. Each line is stripped of leading
+        or trailing whitespace but is otherwise unmolested.
+
+        :return: Entries in the route priority table from `netstat -rn`
+        :rtype: list(str)
+        """
+        linux_style_route_file = [ "Iface\tDestination\tGateway\tFlags\tRefCnt\tUse\tMetric\tMask\tMTU\tWindow\tIRTT" ]
+
+        cmd = "netstat -rn"
+        ret, netstat_output = shellutil.run_get_output(cmd)
+        if ret:
+            raise OSUtilError("Cannot read route table [{0}]".format(netstat_output))
+        netstat_output = netstat_output.split("\n")
+        if len(netstat_output) < 3:
+            return linux_style_route_file
+        netstat_output = netstat_output[3:]
+        # Parse the Netstat -RN header line
+        n_columns = 0
```

Comment:
Could you add a comment with a sample output of `netstat -rn` is to make it easier for the reader to understand the parsing logic

Reply:
Exmples added as requested in e9d7a64